plugins/aep-target-configuration/plugin.json

```diff
@@ -1,10 +1,10 @@
 {
-    "category": "AEP Target Configuration",
-    "displayName": "Adobe Target AEP SDK Extension Configuration",
-    "description": "Verifies the Adobe Target AEP SDK Extension has been configured.",
+    "category": "Adobe Target Configuration",
```

Comment:
Adobe Target or AEP SDK Configuration

Reply:
Sent the category names to PM and he went through and ok with the latest updated names.